numpy/lib/function_base.py

```diff
@@ -1482,42 +1482,48 @@ def angle(z, deg=False):
     return a
 
 
-def _unwrap_dispatcher(p, discont=None, axis=None):
+def _unwrap_dispatcher(p, discont=None, axis=None, *, period=None):
     return (p,)
 
 
 @array_function_dispatch(_unwrap_dispatcher)
-def unwrap(p, discont=pi, axis=-1):
+def unwrap(p, discont=None, axis=-1, *, period=2*pi):
     """
```

Comment:
```suggestion
    r"""
```
will fix CI

Reply:
Thanks for the commit! sorry for the late response, I was on holiday.